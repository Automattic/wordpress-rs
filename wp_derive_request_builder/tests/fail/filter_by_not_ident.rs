#[derive(wp_derive_request_builder::WpDerivedRequest)]
enum UsersRequest {
    #[contextual_get(url = "/users", output = Vec<SparseUser>, "filter_by" = SparseUserField)]
    List,
}

fn main() {}
