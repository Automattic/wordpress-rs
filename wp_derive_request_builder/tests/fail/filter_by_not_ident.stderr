error: Expecting key value pairs (url = "", params = FooParam, output = FooOutput, filter_by = FooSparseField)
 --> tests/fail/filter_by_not_ident.rs:3:64
  |
3 |     #[contextual_get(url = "/users", output = Vec<SparseUser>, "filter_by" = SparseUserField)]
  |                                                                ^^^^^^^^^^^
